src/k8s-engine/k8s.ts

```diff
@@ -165,7 +168,7 @@ export interface KubernetesBackend extends events.EventEmitter {
 export function factory(): KubernetesBackend {
   switch (os.platform()) {
   case 'darwin':
-    return new Hyperkit();
+    return new LimaBackend();
```

Comment:
Nothing is calling `Hyperkit()` anymore. Can it be removed, or is there a chance we'll need it in the future. And if we don't need it anymore, can we also drop scripts/download/hyperkit.mjs and src/k8s-engine/hyperkit.ts ?